<!-- DESCRITPION -->

<launch>
	<node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
	<param name="destination_frame" value="robot_base_chapa"/> <!--TODO-->
	<param name="cloud_destination_topic" value="/merged_cloud"/>
	<param name="scan_destination_topic" value="/scan_multi"/>
	<!-- <param name="laserscan_topics" value ="/scansx /scandx" />  LIST OF THE LASER SCAN TOPICS TO SUBSCRIBE -->
	<param name="laserscan_topics" value ="/front_sick/scan /rear_sick/scan" /> <!-- TODO -->
	<param name="angle_min" value="-3.14"/>
	<param name="angle_max" value="3.14"/>
	<param name="angle_increment" value="0.0058"/>
	<param name="scan_time" value="0.0333333"/>
	<param name="range_min" value="0.30"/>
	<param name="range_max" value="50.0"/>
	</node>
</launch>
